tests/constraints.txt

```diff
@@ -36,7 +36,7 @@ tomli == 2.0.1
 
 # Undeclared extras to "unblock" extra features
 
-shapely == 1.7.0
-# Warning: binary package only available up to python <= 3.8.
+shapely == 1.8.0
+# Warning: binary package only available up to python <= 3.9
```

Comment:
Is this message correct?

Reply:
TBH, the placement of that comment seems wrong, as the statement is valid both for numpy and for shapely, therefore I'd move it above the two.

The sentence is not precise because for example shapely 1.8 supports Python 3.10 too. However I think it's important that `3.9` is in the comment, because this allows to grep for it when it's time to remove support from a Python version.

Therefore I propose something like:

```
# Undeclared extras to "unblock" extra features
#
# Warning: the versions specified for these packages are the oldest versions
# offering a binary package for the oldest Python version we support.
#
# When the minimum supported Python version is increased, these dependencies
# might need to be updated.
#
# Grep help: the current minimum supported version is Python 3.9.
shapely == 1.8.0
numpy == 1.20.0
```

Reply:
Ok, I'll just copypaste it.